lib/adapters/class-adapter.php

```diff
@@ -303,6 +310,111 @@ public function add_term_aggregation( string $label, string $term_field, string
 		);
 	}
 
+	/**
+	 * A callback to filter es args.
+	 * Adds phrase matching to the request args if it is enabled.
+	 *
+	 * @param array $es_args The request args to be filtered.
+	 * @return array The filtered request args.
+	 */
+	public function add_phrase_matching_to_es_args( array $es_args ): array {
+		if ( ! $this->get_enable_phrase_matching() ) {
+			return $es_args;
+		}
+
+		$search = get_query_var( 's' );
```

Comment:
Is it intentional that only search global queries is handled? I'm thinking of custom requests with `WP_Query` or the REST API where this query var is not set globally.

Reply:
This is intentional as per the use cases that I have been asked to cater to so far.

Reply:
I see! Is it possible to document this somehow in the wiki?

Reply:
I will add a wiki entry as soon as this is merged in :)